portafly/portafly_api.yml

```diff
@@ -0,0 +1,102 @@
+---
+openapi: 3.0.2
+info:
+  title: PortaFly
+  version: 1.0.0
+  description: |
+    A tuned API for PortaFly, a Patternfly SPA consuming 3scale Porta API MGMT solution.
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /developer/accounts:
+    summary: A list of developer accounts
+    description: A list of developer accounts.
+    get:
+      responses:
+        "200":
+          $ref: '#/components/responses/developerAccountList'
+      operationId: developerAccountList
+      summary: Get developer account list
+    parameters:
+    - name: page
+      description: Page in the paginated list. Defaults to 1.
+      schema:
+        type: integer
+      in: query
+      required: false
+    - name: per_page
+      description: Number of results per page. Default and max is 500.
+      schema:
+        type: integer
+      in: query
+      required: false
+    - name: state
+      description: Account state.
+      schema:
+        type: string
+      in: query
+components:
+  schemas:
+    developerAccountData:
```

Comment:
Why suffixing with `Data`?

Can't it be just `DeveloperAccount`

Reply:
Also I am wondering if Account is better? Or is it inherited?

Account  -> DeveloperAccount
Account  -> ProviderAccount

Reply:
You are right, no need to sufix with `Data` and should start with Capital letter. I'd say let's leave it as `DeveloperAccount` and when we need to define the `Account` data type I change it. Let's do it a bit by bit. What do you think?